3.analyze_data/compare_raw_mitocheckjump.ipynb

```diff
@@ -0,0 +1,696 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    for feature in all_features:
</pre><p>Could also use list or dictionary comprehension</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/JUMP-single-cell/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I agree! FWIW, I opted to use a regular loop to increase readability. Sometimes I get confused with list comprehension, which decreases understandability.